packages/shared/tailwind.config.js

```diff
@@ -238,7 +238,21 @@ module.exports = {
     },
   },
   // eslint-disable-next-line global-require
-  plugins: [caret, typography, buttons],
+  plugins: [
+    caret, 
+    typography, 
+    buttons,
+    function ({ addUtilities }) {
+      const newUtilities = {
+        '.bg-clip-text': {
```

Comment:
This is already a tailwind class right?
https://tailwindcss.com/docs/background-clip

Reply:
Yeah, but the color fill doesn't match our requirements. Should we maybe let go of the plugin and add the missing CSS property on the base.css input selector?

Reply:
@sshanzel Thanks, I've removed `bg-clip-text` from the plugin. As it is already exists, I used it and applied `var(--theme-label-primary)` to the `input` selector on base.css, so now it is consistent. Also, I am sure there must be a way to add `bg-clip-text` on the global level to the `input` selector instead of adding separate.

Reply:
Kindly see this comment https://github.com/dailydotdev/apps/pull/2533#discussion_r1463008425.

You are almost there! 🚀